examples/SpringHibernateBalancedReaderOneDataSourceExample/src/main/resources/application.yml

```diff
@@ -0,0 +1,36 @@
+spring:
+  profiles.active: development
+  jpa:
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+
+logging:
+  level:
+    software:
+      amazon:
+        jdbc: INFO
+        jdbc.states: INFO
+    example: TRACE
+
+---
+spring:
+  profiles: development
```

Comment:
I see a warning saying `profiles` is redacted 
![image](https://github.com/awslabs/aws-advanced-jdbc-wrapper/assets/64801825/c71fcca1-cea6-41e4-9fee-31af2ba78051)